src/widgets/text.rs

```diff
@@ -84,13 +84,15 @@ pub fn Text(props: TextProps) {
             ((0.0, 0.0), (0.0, 0.0))
         };
 
+
+
     if should_render {
         let render_command = RenderCommand::Text {
             content: content.clone(),
             size,
             parent_size,
             line_height: line_height.unwrap_or(size * 1.2),
-            font: font_name.clone().unwrap_or("Roboto".into()),
+            font: font_name.clone().unwrap_or(default_style),
         };
```

Comment:
I wonder if maybe we could make `RenderCommand::Text` only take the content. Then in `WidgetManager::recurse_node_tree_to_build_primitives(...)` we can actually resolve the styles and get other metadata to build the full primitive.

That might not be super optimal though since it would happen every render. 🤔